docs/admin/provisioners.md

```diff
@@ -47,50 +47,91 @@ the [Helm example](#example-running-an-external-provisioner-with-helm) below.
-  corresponding workspaces) with matching tags.
-
-  ```shell
-  coder provisionerd start \
-    --tag environment=on_prem \
-    --tag data_center=chicago
-
-  # In another terminal, create/push
-  # a template that requires this provisioner
-  coder templates push on-prem \
-    --provisioner-tag environment=on_prem
-
-  # Or, match the provisioner exactly
-  coder templates push on-prem-chicago \
-    --provisioner-tag environment=on_prem \
-    --provisioner-tag data_center=chicago
-  ```
-
-  > At this time, tagged provisioners can also pick jobs from untagged
-  > templates. This behavior is
-  > [subject to change](https://github.com/coder/coder/issues/6442).
-
-- **User provisioners** can only pick up jobs from user-tagged templates. Unlike
-  the other provisioner types, any Coder user can run user provisioners, but
-  they have no impact unless there is at least one template with the
-  `scope=user` provisioner tag.
-
-  ```shell
-  coder provisionerd start \
-    --tag scope=user
-
-  # In another terminal, create/push
-  # a template that requires user provisioners
-  coder templates push on-prem \
-    --provisioner-tag scope=user
-  ```
+> Provisioners have two important tags: `scope` and `owner`. Coder sets these
+> tags automatically.
+
+### Organization-Scoped Provisioners
+
+**Organization-scoped Provisioners** can pick up build jobs created by any user.
+These provisioners always have tags `scope=organization owner=""`.
+
+```shell
+coder provisionerd start
+```
+
+### User-scoped Provisioners
+
+**User-scoped Provisioners** can only pick up build jobs created from
+user-tagged templates. User-scoped provisioners always have tags
+`scope=owner owner=<uuid>`. Unlike the other provisioner types, any Coder user
+can run user provisioners, but they have no impact unless there is at least one
+template with the `scope=user` provisioner tag.
+
+```shell
+coder provisionerd start \
+  --tag scope=user
+
+# In another terminal, create/push
+# a template that requires user provisioners
+coder templates push on-prem \
+  --provisioner-tag scope=user
+```
+
+### Provisioner Tags
+
+You can use **provisioner tags** to control which provisioners can pick up build
+jobs from templates (and corresponding workspaces) with matching tags.
+
+```shell
+coder provisionerd start \
+  --tag environment=on_prem \
+  --tag data_center=chicago
+
+# In another terminal, create/push
+# a template that requires this provisioner
+coder templates push on-prem \
+  --provisioner-tag environment=on_prem
+
+# Or, match the provisioner exactly
+coder templates push on-prem-chicago \
+  --provisioner-tag environment=on_prem \
+  --provisioner-tag data_center=chicago
+```
+
+A provisioner can run a given build job if one of the below is true:
+
+1. The provisioner and job tags are both organization-scoped and both have no
+   additional tags set,
+1. The set of tags of the build job is a subset of the set of tags of the
```

Comment:
This makes it sound like if the job is organization scoped with no further tags it can run on any org-scoped provisioner.

I thought such a job can only run on an org-scoped provisioner with no further tags?

The way I would explain it is that `scope` and `owner` are implicit tags.  

An job with no explicit tags can only be run on a provisioner with no explicit tags. This way you can introduce tagging into your deployment without disrupting existing provisioners and jobs.

If a job has any explicit tags, it can only run on a provisioner with those explicit tags (the provisioner could have additional tags).

Reply:
https://github.com/coder/coder/pull/12347